quests/challenge-github-app/GitHubApp/Screens/List/ListViewController.swift

```diff
@@ -37,18 +39,21 @@ final class ListViewController: UIViewController {
         self.navigationController?.navigationBar.prefersLargeTitles = true
         self.navigationItem.title = "Repositories"
         self.navigationItem.searchController = searchBar
-        self.navigationItem.hidesSearchBarWhenScrolling = false
-        self.navigationItem.searchController?.searchBar.delegate = self
-       
+        searchBar.searchBar.delegate = self
+      
     }
 
     override func viewDidAppear(_ animated: Bool) {
         
-        service.fetchList(user: "GustavoFernandesBatista") { result in
+    }
+    
+    private func fetchRepos(user: String){
+        self.service.fetchList(user) { result in
             switch result {
             case .success(let repository):
-                print(repository.count)
-                
+                DispatchQueue.main.async {
```

Comment:
Boa! Sempre que for fazer alterações de UI de outra thread, devemos executar na thread principal (main).